pom.xml

```diff
@@ -120,13 +120,13 @@
 
     <dependency>
       <groupId>io.vertx</groupId>
-      <artifactId>vertx-rx-java2-gen</artifactId>
+      <artifactId>vertx-rx-java3-gen</artifactId>
```

Comment:
Lets us not remove rxjava2. We should generate rxifed code for both rxjava2 and rxjava3 so that the clients can use however they want